_data/accessibility-tests/character-count.yml

```diff
@@ -0,0 +1,81 @@
+title: Character count
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain the field's input requirements.
+    summary_additional: |
+      The text field has a visible label above it to indicate its purpose.
+      For example, "Enter your full name."
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.3.5
+  - summary: Users can see information about the number of characters remaining.
```

Comment:
Criterion `1.3.5` is listed as `pass` in the checklist but your use of `conditional` seems more accurate. Should we update the checklist to reflect this, or is there a reason we haven’t?

Reply:
I updated the spreadsheet to reflect this. 

For reference, this topic was discussed in [this comment thread](https://github.com/uswds/uswds-site/pull/2607#discussion_r1556334590).